.vscode/launch.json

```diff
@@ -164,6 +164,38 @@
         "services/validation/test/truffle-example-incorrect.zip"
       ]
     },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Validation service (truffle project - directory)",
+      "skipFiles": [
+        "<node_internals>/**"
+      ],
+      "program": "${workspaceFolder}/services/validation/build/index.js",
+      "preLaunchTask": "tsc: build - services/validation/tsconfig.json",
+      "outFiles": [
+        "${workspaceFolder}/services/validation/build/**/*.js"
+      ],
+      "args": [
+        "services/validation/test/truffle-example"
```

Comment:
This directory is actually missing .json files and should be "truffle-example-incorrect" so this returns to me that I am missing metadata.json file.

Reply:
@Jakic007 I think this was caused by .gitignore specifying everything called build to be ignored. I'll fix this.